source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -137,6 +164,28 @@ content: |
                const extraOptions = {
                  mongocryptdBypassSpawn: true,
                }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 2
+      
+           extra_options = {
+              'mongocryptd_spawn_path': '/usr/local/bin/mongocryptd'
+           }
+
+        .. admonition:: Encryption Binary Daemon
```

Comment:
I like this, but can you update the Node and Java examples so that they do not re-declare `extraOptions`?

Reply:
I'd love to do that, I just don't want to hold python for this. I created a ticket for doing this: https://jira.mongodb.org/browse/DOCSP-7332 for next sprint